Sandbox/Sandbox/LoginWithFidoViewController.swift

```diff
@@ -0,0 +1,34 @@
+import UIKit
+import IdentitySdkCore
+import PromiseKit
+import CryptoSwift
+
+class LoginWithFidoViewController: UIViewController{
+ 
+    @IBOutlet weak var emailText: UITextField!
+    let scopes = ["openid", "email", "profile", "phone", "full_write", "offline_access"]
+
+    override func viewDidLoad() {
+           super.viewDidLoad()
+          
+        AppDelegate.reachfive().initialize().onComplete { _ in }
```

Comment:
Q: what does `.onComplete { _ in }` do here?

Reply:
This line is used to initialize the reachfive object

Reply:
```suggestion
        AppDelegate.reachfive().initialize()
```